Zend/zend.c

```diff
@@ -1484,6 +1499,17 @@ ZEND_API ZEND_COLD void zend_error_zstr_at(
 				ex->opline = opline;
 			}
 		}
+	} else if (EG(error_backtrace_recording) & type) {
+		if (!Z_ISUNDEF(EG(error_backtrace))) {
+			zval_ptr_dtor(&EG(error_backtrace));
+		}
```

Comment:
Checking `!Z_ISUNDEF` is not necessary here. `zval_ptr_dtor` will simply be a noop. Likewise in the other files.